### PR TITLE
Run SMT only on latest EVM version.

### PR DESCRIPTION
**Pull request by chriseth**
I was not able to find out how to use decorators with a manually-created test graph.

### CODE REVIEW

File: .circleci/soltest_all.sh [hunk 1]
```diff
@@ -32,6 +32,6 @@ EVM=istanbul OPTIMIZE=1 ABI_ENCODER_V2=1 ${REPODIR}/.circleci/soltest.sh
 
 for OPTIMIZE in 0 1; do
     for EVM in homestead byzantium constantinople petersburg istanbul; do
-        EVM=$EVM OPTIMIZE=$OPTIMIZE BOOST_TEST_ARGS="-t !@nooptions" ${REPODIR}/.circleci/soltest.sh
+        EVM=$EVM OPTIMIZE=$OPTIMIZE BOOST_TEST_ARGS="--no-smt -t !@nooptions" ${REPODIR}/.circleci/soltest.sh
```
Comment (leonardoalt) [inline .circleci/soltest_all.sh:35]: I think the way CI does is by setting `--no-smt` to `SOLTEST_ARGS` instead

**Review by leonardoalt — CHANGES_REQUESTED** [review]
Only one comment

**Comment by ekpyron** [comment]
It should be the optional second argument to ``test_suite::add``... But maybe creating the decorators themselves is a hassle... EDIT: not exactly that, but I got it in the new PR.

**Comment by ekpyron** [comment]
I think I might know how to add the label, let me try for a second...

**Pull request closed without merging**